src/main/java/com/mercadolibrewannabe/utils/enums/Rating.java

```diff
@@ -1,21 +1,21 @@
 package com.mercadolibrewannabe.utils.enums;
 
-public enum RatingEnum {
+public enum Rating {
 	ONE(1), TWO(2), THREE(3), FOUR(4), FIVE(5);
 
 	private final Integer value;
 
-	private static final RatingEnum[] valuesArray = values();
+	private static final Rating[] valuesArray = values();
 
-	RatingEnum (Integer i) {
+	Rating (Integer i) {
 		this.value = i;
 	}
 
 	public Integer getValue () {
 		return value;
 	}
 
-	public static RatingEnum byValue(Integer value) {
+	public static Rating byValue(Integer value) {
 		return valuesArray[value - 1];
```

Comment:
Pq não usar o método values direto aqui?

Reply:
Pra não ficar processando o Enum para array toda a vez que chamar o método `byValue`. Armazeno como um final static. É melhor ou to viajando?

Reply:
relaxa com processar sempre isso aí... chamar um código várias vezes tende a não ser gargalo, o jit ajuda. Para este tipo de coisa ser gargalo a gente precisa estar num cenário de guerra total dentro do código.